Content.Shared/Doors/Components/DoorComponent.cs

```diff
@@ -62,6 +62,12 @@ public sealed class DoorComponent : Component, ISerializationHooks
     [DataField("emagDuration")]
     public readonly TimeSpan EmagDuration = TimeSpan.FromSeconds(0.8f);
 
+    /// <summary>
+    ///     Pry modifier for a powered door.
+    /// </summary>
+    [DataField("poweredPryModifier")]
+    public readonly float PoweredPryModifier = 2.5f;
```

Comment:
this should be significantly longer. IMO prying open a powered door should take at least 8-10 seconds--these shouldn't be at the same power level as an emag with infinite charges

Reply:
also this shouldn't be a door thing this should be an airlock thing?